atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/KvsGetRowsColumnRangeBenchmarks.java

```diff
@@ -40,53 +40,49 @@
 @State(Scope.Benchmark)
 public class KvsGetRowsColumnRangeBenchmarks {
 
+    public static final int EXPECTED_NUM_CELLS = WideRowsTable.NUM_ROWS * WideRowsTable.NUM_COLS_PER_ROW;
+    public static final List<byte[]> ROWS = IntStream
+            .rangeClosed(0, WideRowsTable.NUM_ROWS - 1)
+            .mapToObj(WideRowsTable::getRow)
+            .collect(Collectors.toList());
+    public static final int CORRECT_CELL_BATCH_HINT = 10000;
+    public static final int INCORRECT_CELL_BATCH_HINT = 10017;
+
     @Benchmark
-    @Warmup(time = 5, timeUnit = TimeUnit.SECONDS)
-    @Measurement(time = 45, timeUnit = TimeUnit.SECONDS)
+    @Warmup(time = 16, timeUnit = TimeUnit.SECONDS)
```

Comment:
if we wanted to be cautious we would not modify the timings for tests that were also refactored.  This would isolate changes.